autoload/clang_format.vim

```diff
@@ -209,6 +209,7 @@ function! clang_format#replace(line1, line2)
     let sel_save = &l:selection
     let &l:selection = 'inclusive'
     let [save_g_reg, save_g_regtype] = [getreg('g'), getregtype('g')]
+    let [save_unnamed_reg, save_unnamed_regtype] = [getreg(''), getregtype('')]
```

Comment:
I think spcifying `''` to `getreg` or `setreg` or something is undocumented (at least I could not find the document for it). It looks that `v:register` is better.

```
let [save_unnamed_reg, save_unnamed_regtype] = [getreg(v:register), getregtype(v:register)]

" ...

call setreg(v:register, save_unnamed_reg, save_unnamed_regtype)
```

Could you update changes and confirm the behavior?